client-docs/README.md

```diff
@@ -198,26 +216,26 @@ type VersionId = Opaque<string>;
 
 type PartialSyncState = {
   /** Number of blocks we have locally */
-  have: number,
+  have: number;
   /** Number of blocks we want from connected peers */
-  want: number,
+  want: number;
   /** Number of blocks that connected peers want from us */
-  wanted: number,
+  wanted: number;
   /** Number of blocks missing (we don't have them, but connected peers don't have them either) */
-  missing: number,
+  missing: number;
   /** Is there data available to sync? (want > 0 || wanted > 0) */
-  dataToSync: boolean,
+  dataToSync: boolean;
   /** Are we currently syncing? */
-  syncing: boolean,
-}
+  syncing: boolean;
+};
 
 type SyncState = {
   /** State of initial sync (sync of auth, metadata and project config) */
-  initial: PartialSyncState,
+  initial: PartialSyncState;
   /** State of data sync (observations, map data, photos, audio, video etc.) */
-  data: PartialSyncState,
+  data: PartialSyncState;
   /** Number of connected peers */
-  connectedPeers: number
+  connectedPeers: number;
```

Comment:
These should have been caught by prettier....we should a pre-commit step to this repo